Marlin/src/module/probe.cpp

```diff
@@ -293,7 +293,7 @@ FORCE_INLINE void probe_specific_action(const bool deploy) {
         host_prompt_do(PROMPT_USER_CONTINUE, PSTR("Stow Probe"), PSTR("Continue"));
       #endif
       #if ENABLED(EXTENSIBLE_UI)
-        ExtUI::onUserConfirmRequired(PSTR("Stow Probe"));
+        ExtUI::onUserConfirmRequired_P(PSTR("Stow Probe"));
```

Comment:
@InsanityAutomation — @marcio-ao — It sounds like you should confer on whether this is the proper use case for `onUserConfirmRequired`.